pkg/api/types.go

```diff
@@ -602,6 +638,8 @@ type LiteralTestStep struct {
 	// Cli is the (optional) name of the release from which the `oc` binary
 	// will be injected into this step.
 	Cli string `json:"cli,omitempty"`
+	// Observers are the observers that should be running
```

Comment:
Nit: It would be nice to write one sentence about what an oberver is here

Reply:
I expect to add an architecture page to the docs site for this